app/View/Email/ReservationTemplate.htm

```diff
@@ -0,0 +1,17 @@
+<html>
+    <body>
+        <p>Dobrý den,</p>
+
+        <p>
+            Děkujeme Vám za rezervaci lístků na
+            <b>Reprezentační ples Univerzity Tomáše Bati 2023</b>.
+        </p>
+        <p>
+            Máte {{count}} míst u stolu č.{{table}} . Vaše rezervace vyprší za 3
```

Comment:
wouldn't be better to define the variable `expiryDate` to not use hard-coded 3 days?

Reply:
For now, i will just put it inside variable, later, it will be customizable in administration, will change that with new template
data